paddle/framework/net.cc

```diff
@@ -3,18 +3,24 @@
 namespace paddle {
 namespace framework {
 
-PlainNet::PlainNet(const NetDesc& def) {}
+void PlainNet::AddOp(const OpDesc& desc) {
+  std::unique_ptr<OperatorBase> op(OpRegistry::CreateOp(desc));
+  ops_.push_back(std::move(op));
```

Comment:
emplace_back is better. 
[emplace_back vs push_back](https://stackoverflow.com/questions/4303513/push-back-vs-emplace-back)